projects/graph/src/graph.py

```diff
@@ -6,4 +6,26 @@
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        self.verts = {}
+    
+    def add_vertex(self, vert):
+        if not vert in self.verts:
+            self.verts[vert] = set()
+        else:
+            print(f"Vert {vert} already exists")
+    
+    def add_edge(self, vert, edge):
+        if vert in self.verts:
+            self.verts[vert].add(edge)
```

Comment:
Alberto, I like how efficient your implementation is here. The simplicity here will make other parts of your project easier to work with in the coming days.